src/PIL/Image.py

```diff
@@ -538,6 +538,10 @@ def __init__(self):
         self.readonly = 0
         self.pyaccess = None
         self._exif = None
+        if "is_animated" not in dir(self):
+            self.is_animated = False
+        if "n_frames" not in dir(self):
+            self.n_frames = 1
```

Comment:
These properties can't just be always set regardless of whether or not they are already present, because for [some formats, they are property decorators without setters](https://github.com/python-pillow/Pillow/blob/4634eafe3c695a014267eefdce830b4a825beed7/src/PIL/GifImagePlugin.py#L92-L102).